dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisPipeline.java

```diff
@@ -2142,12 +2142,21 @@ public String get(String key, String val) {
 
```

Comment:
Changes look good to me. I am wondering if this function is straightforward to be added too? otherwise go ahead and merge it.